src/index.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import './sass/style.scss'
-import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Good change here. Now the Sass compiler takes care of classes clash.

Reply:
Thanks!